.meshrc.yaml

```diff
@@ -57,6 +57,12 @@ sources:
             method: GET
             responseSchema: ./json-schemas/mngsWorkflowResults.json
             responseTypeName: MngsWorkflowResults
+          - type: Query
+            field: CoverageVizSummary
+            path: /samples/{args.sampleId}/coverage_viz_summary
+            method: GET
+            responseSchema: ./json-schemas/coverageVizSummary.json
+            responseTypeName: CoverageVizSummary            
```

Comment:
I think coverage viz summary may also be available at the endpoint `/pub/[snapshotShareId]`.  It [appears to call the samples controller](https://github.com/chanzuckerberg/czid-web-private/blob/a4dd2a773881ee9f2b5a5707225f15280e052903/app/controllers/snapshot_samples_controller.rb#L149-L152). but should we add a config to federate that as well?

Reply:
ah wait, that ticket was created for the entity it looks like. Will add this endpoint

Reply:
Looking into this a bit further, I am not sure why that endpoint is part of the ticket. It renders a template

Reply:
The endpoint at `GET /pub/:share_id/samples/:id/coverage_viz_summary` does not need to be federated since the implementation of it calls the same method (via inheritance) as the one federated here

Reply:
Federated public endpoint